schema/claim.schema.json

```diff
@@ -44,17 +52,13 @@
     "custom": {
       "$comment": "reserved for custom extensions"
     },
-    "modified": {
-      "description": "The date last modified, as an ISO-8601 Extended Format date string, as specified in the ECMAScript standard",
-      "type": "string"
-    },
     "installation": {
       "description": "the name of the installation",
       "type": "string"
     },
-    "outputs": {
-      "description": "key/value pairs of output name to output value.",
-      "type": "object"
+    "modified": {
```

Comment:
This `modified` property should move up into the `result` definition, right?

Reply:
Thanks, yes my git-fu went sideways this morning.